.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,42 +38,61 @@ body:
-      Please do not add code examples or error messages as screenshots, but as text, this helps searching for issues and testing the code. If you are reporting a bug in the editor interface, like the script editor, please provide both a screenshot *and* the text of the code to help with testing.
+      Describe your issue briefly.
+
+      You can include images or videos with drag and drop, and format code
+      blocks or logs with <code>```</code> tags, on separate lines before and
+      after the text. (Use <code>```gdscript</code> to add GDScript syntax
+      highlighting.)
+
+      > [!IMPORTANT]
+      >  Please do **not** add code examples or error messages as screenshots
   validations:
     required: true
 
 - type: textarea
   attributes:
     label: Steps to reproduce
     description: |
-      List of steps or sample code that reproduces the issue. Having reproducible issues is a prerequisite for contributors to be able to solve them.
-      If you include a minimal reproduction project below, you can detail how to use it here.
+      List of steps or sample code that reproduces the issue. Contributors need
+      to reproduce issues in order to fix them.
   validations:
     required: true
 
 - type: textarea
   attributes:
     label: Minimal reproduction project (MRP)
     description: |
-      - A small Godot project which reproduces the issue, with no unnecessary files included. Be sure to not include the `.godot` folder in the archive (but keep `project.godot`).
-      - Having an MRP is very important for contributors to be able to reproduce the bug in the same way that you are experiencing it. When testing a potential fix for the issue, contributors will use the MRP to validate that the fix is working as intended.
-      - If the reproduction steps are not project dependent (e.g. the bug is visible in a brand new project), you can write "N/A" in the field.
-      - Drag and drop a ZIP archive to upload it (max 10 MB). **Do not select another field until the project is done uploading.**
-      - **Note for C# users:** If your issue is *not* C#-specific, please upload a minimal reproduction project written in GDScript. This will make it easier for contributors to reproduce the issue locally as not everyone has a .NET setup available.
+      Contributors will use the MRP to validate that the fix is working as
+      intended.
+      - Drag and drop a ZIP archive containing a small Godot project which
+        reproduces the issue to upload it (max 10 MB). **Do not select another
+        field until the project is done uploading.**
+      - Make sure that the project contains the minimal amount of assets or
+        code.
+      - Be sure to not include the `.godot` folder in the archive (but keep
```

Comment:
Note that some bugs reproduce only when the .godot folder is the same (cache bugs)

Reply:
Again, we must be terse. Unfortunately, `.godot` folder is rather hefty, and there's no native way to seamlessly export a project folder without `.godot`.
There are always exceptions like these that require steps beyond the single execution of the project.

Reply:
But the cache bug still needs to be reproducible, isn't it?